src/Template/Element/sentences/sentence_menu.ctp

```diff
@@ -1,6 +1,8 @@
-<div class="menu-wrapper" sentence-menu flex="{{vm.isMenuExpanded ? '100' : 'none'}}" 
+<div class="menu-wrapper" sentence-menu flex="{{vm.isMenuExpanded ? '100' : 'none'}}"
      ng-init="vm.initMenu(<?= (int)$expanded ?>, vm.sentence.permissions)">
-    <div class="menu" layout="row" layout-align="space-between center">
+    <div class="menu" layout="row" layout-align="space-between center"
+         ng-controller="ReviewsController as reviewCtrl"
```

Comment:
The reason why you have to put the `ng-controller` directive here is that the we use `ng-if` for deciding whether we show the expanded menu or not. But `ng-if` deletes and recreates an element every time the conditions change and thus you call `reviewCtrl.initSentence` again which initializes the sentence with the old data (the original one received from the server) and consequently we loose the changes the user already made before.

Using `ng-show` instead you could move the `ng-controller` back where it was before and then check in `reviewCtrl.initSentence` whether `vm.sentence` is already initialized.